BROTLI_HUFFMAN_CODE_FAST_LOAD is only enabled on armv7/armv8_32, and it doesn't
actually compile - likely due to -std=gnu89.

In file included from ./brotli/c/dec/decode.c:20:
./brotli/c/dec/decode.c: In function 'PreloadSymbol':
./brotli/c/dec/huffman.h:83:47: error: ISO C90 forbids mixed declarations and code [-Werror=declaration-after-statement]
   83 | #define BROTLI_HC_MARK_TABLE_FOR_FAST_LOAD(H) uint32_t __fastload_##H = (*H)
      |                                               ^~~~~~~~

diff --git a/brotli/c/dec/huffman.h b/brotli/c/dec/huffman.h
index 50360962c..1d1d6beab 100644
--- a/brotli/c/dec/huffman.h
+++ b/brotli/c/dec/huffman.h
@@ -28,7 +28,7 @@ extern "C" {
 
 #define BROTLI_HUFFMAN_MAX_CODE_LENGTH_CODE_LENGTH 5
 
-#if ((defined(BROTLI_TARGET_ARMV7) || defined(BROTLI_TARGET_ARMV8_32)) && \
+#if 0 && ((defined(BROTLI_TARGET_ARMV7) || defined(BROTLI_TARGET_ARMV8_32)) && \
   BROTLI_GNUC_HAS_ATTRIBUTE(aligned, 2, 7, 0))
 #define BROTLI_HUFFMAN_CODE_FAST_LOAD
 #endif
